src/vorta/i18n/__init__.py

```diff
@@ -46,7 +44,11 @@ def translate(self, context, text, disambiguation=None, n=-1):
 def init_translations(app):
     global application, translator, locale  # if we don't keep a reference on these, it stops working. pyqt bug?
     application = app
-    translator = VortaTranslator()
+    if trans_scale == 100:
+        translator = QTranslator()
+    else:
+        translator = VortaTranslator()  # Doesn't fall back to EN on empty strings.
+        
```

Comment:
would be interesting to find out why this changes observed behaviour.

Reply:
It would. Some classes are just not meant to be subclassed by mere mortals.

Reply:
nitpick:
```
        translator = QTranslator() if trans_scale == 100 else VortaTranslator() 
```